docs/marionette.application.md

```diff
@@ -160,6 +160,21 @@ allows you to provide extra configuration for various parts of your app througho
 initialization sequence.
 
 ```javascript
+var App = Mn.Application.extend({
+  initialize: function(options) {
+    console.log('Initialize');
+  },
+
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(options.data); 
```

Comment:
@denar90 Apologies, I was a bit unclear - I think using `options` here is perfectly correct as it demonstrates the use of arguments as @rmc33 points out. Removing it earlier was just to follow the style guide from `eslint`.